python/enthic/download_from_INPI.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+"""
+========================================
+Download daily zip files from INPI's FTP
+========================================
+
+Coding Rules:
+
+- Snake case for variables.
+- Only argument is configuration file.
+- No output or print, just log and files.
+"""
+
+from os.path import dirname, join, basename
+from os import remove
+import subprocess
+from json import load
+from ftplib import FTP_TLS
+from logging import info
+
+from enthic.extract_bundle import process_daily_zip_file
+
+
+################################################################################
+# READ CONFIGURATION
+with open(join(dirname(__file__), "configuration.json")) as json_configuration_file:
+    CONFIG = load(json_configuration_file)
+
+def add_data_to_database(file_path):
+    """
+        Add data contained in the given file into MySQL database
+        then delete temporary files if it's a success
+
+        :param file_path: daily INPI file to add to database
+    """
+    # extract data from xml files to csv files
+    process_daily_zip_file(file_path)
+
+    # Load new data into database
+    result = subprocess.run(["../sh/database-filling.sh", CONFIG["mySQL"]["enthic"]["password"]], stdout=subprocess.PIPE, check=True)
+    if result.stdout:
+        raise Exception("Following error while loading data from file "
+                        + file_path + " into database : \n"
+                        + result.stdout.decode('utf-8'))
+    else:
+        remove(join(CONFIG['outputPath'], CONFIG['tmpBundleFile']))
+        remove(join(CONFIG['outputPath'], CONFIG['identityFile']))
+        remove(join(CONFIG['outputPath'], CONFIG['metadataFile']))
+        remove(file_path)
+
+
+def exploreAndProcessFTPFolder(folderToExplore) :
```

Comment:
definition name is not compliant with PEP8. Method are to be name in snake_case style.